mllm/engine/HpcThreadPool.hpp

```
@@ -0,0 +1,75 @@
+
+#pragma once
+
+#include <mutex>
+#include <thread>
+#include <vector>
+#include <atomic>
+#include <memory>
+#include <functional>
+#include <condition_variable>
+
+#define MLLM_HPC_THREAD_POOL_TASK_LIMITS 2
```

Clang (14.0.6)

[error] macro 'MLLM_HPC_THREAD_POOL_TASK_LIMITS' used to declare a constant; consider using a 'constexpr' constant

(cppcoreguidelines-macro-usage,-warnings-as-errors)